server/plugin/utils.go

```diff
@@ -40,14 +40,16 @@ func getYourAssigneeSearchQuery(username string, orgs []string) string {
 	return buildSearchQuery("is:open assignee:%v archived:false %v", username, orgs)
 }
 
-func getIssuesSearchQuery(org, searchTerm string) string {
+func getIssuesSearchQuery(searchValue, searchTerm string) string {
 	query := "is:open is:issue archived:false %v %v"
-	orgField := ""
-	if len(org) != 0 {
-		orgField = fmt.Sprintf("org:%v", org)
+	searchField := ""
+	if len(searchValue) != 0 {
+		searchField = fmt.Sprintf("org:%v", searchValue)
+	} else { // get all the issues which involve the user in case no organisaitonal lock is set 
+		searchField = fmt.Sprintf("involves:@me")
```

Comment:
What if the user is commenting for the first time on that PR?

Reply:
@raghavaggarwal2308

Reply:
I think this was implemented before. We are fetching the issues globally on the basis of the search param.